src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form name="form">
+      <fieldset class="perInfo fieldset" name="Personal">
+        <legend>Personal information</legend>
+
+        <label for="surName" class="input-label">
+          Surname:
+          <input 
+            type="text"
+            name="username" 
+            required
+          >
+        </label>
+
+        <label for="name" class="input-label">
+          Name:
+          <input 
+            id="name"
+            type="text"
+            name="name" 
+            required
+          >
+        </label>
+
+        <label for="age" class="input-label">
+          How old are You?
+          <input 
+            id="age" 
+            class="age"
+            type="number" 
+            name="age" 
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label for="date" class="input-label">
+          Full date of birth:
+          <input 
+            id="date" 
+            type="date"
+            name="date" 
+            required
+          >
+        </label>
+
+        <label for="checkbox" class="input-label">
+          I accept the term of the agreement
+          <input 
+            id="checkbox" 
+            type="checkbox" 
+            name="agreement" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="regist fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="input-label">
+          E-mail:
+          <input 
+            id="email" 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required 
+            multiple
+          >
+        </label>
+
+        <label for="password" class="input-label">
+          Password:
+          <input 
+            id="password" 
+            type="password" 
+            name="password" 
+            minlength="10" 
+            maxlength="20" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fact fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label for="radio" class="input-label">
+          Do you love cats?
+          <input 
+            id="radio"
+            type="radio"
+            name="radio"
+            value="y"
+          > 
+            Yes
+          <input 
+            id="radio1" 
+            type="radio" 
+            name="radio" 
+            value="n"
+          >
+            No
```

Comment:
create separate labels for these inputs. now when i click on `Do you love cats?` it selects the first option (`yes`)